Backpack-SwiftUI/Price/Classes/BPKPrice.swift

```diff
@@ -28,24 +76,24 @@ public struct BPKPrice: View {
         case leading, trailing
     }
     
-    public let price: String
-    public let leadingText: String?
-    public let lineThroughText: String?
-    public let trailingText: String?
-    public let alignment: Alignment
-    public let size: Size
+    private let price: String?
+    private let leadingText: String?
+    private let previousPrice: String?
+    private let trailingText: String?
+    private let alignment: Alignment
+    private let size: Size
     
     public init(
-        price: String,
+        price: String?,
```

Comment:
I think my other comment about the price not having a value was only relevant for UIKit, for SwiftUI handling the value as required if the component requires it would be an expected way to handle it.
@gert-janvercauteren what do you think about this? shall this property be required in SwiftUI and not required in UIKit? (see my previous comment on the uikit class)

Reply:
I think as it does not make sense to have a price when there's no price, in SwiftUI that should be handled by the client.
ie:
```swift
struct MyScreen: View {
  @ObservedObject let priceViewModel: PriceViewModel
  var body: some View {
    if let price = priceViewModel.price {
       BPKPrice(price: price) 
    }
  }
}
```
I think this is the expected API. It would be nice to support it for UIKit, but UIKit cant achieve this because of how it, and the view controllers lifecycle works